docs/content/latest/develop/graphql/hasura.md

```diff
@@ -120,11 +136,10 @@ Click **Add**, and then click **Save**.
 ./bin/ysqlsh
 ```
 
-1. Copy the commands below into the shell and press **Enter**.
+1. Copy the YSQL statements below into the shell and press **Enter**.
 
 ```postgresql
-SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL SERIALIZABLE
```

Comment:
Will be good to check with @m-iancu whether it's okay to remove this statement.

Reply:
Yes, this can be removed now (it's an artifact from before we supported FK checks in `REPEATABLE READ` isolation).